docs/release-notes/february-2023.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: february-2023
+append_help_link: true
+hide_title: true
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+---
+
+# February 2023
+
+## Important update
+
+- Semgrep CLI is now officially renamed to **Semgrep OSS Engine**. Note that Semgrep CLI is an interface, not a product. As an interface, you can use the CLI as an interface for several products. For example the OSS, Semgrep Code, Semgrep Supply Chain.
```

Comment:
```suggestion
- Semgrep CLI is now officially renamed to **Semgrep OSS Engine**. You can use Semgrep CLI for several products, such as Semgrep OSS, Semgrep Code, and Semgrep Supply Chain.
```

My "Semgrep CLI is an interface, not a product" note in the draft was an internal note, because it spells out to "Semgrep Command line interface is an interface" i.e. redundant.

Reply:
I knew it is internal, but have been thinking that it was really cool to include it. Thanks for the suggestion! @s-santillan